dae-alright/src/components/Header/HeaderWraper/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import "./HeaderWraper.scss"
+
+function HeaderWraper(props) {
+    return (
+        <header className="header-wrapper">
+            
```

Comment:
Is this not supposed to call the `render props`? Where did that function go?

Reply:
You said i should delete the component that takes in a renderProp. doing that and having props.renderProp in dis component will produce error. thats why i removed it